### ISSUE TITLE
Display metadata in COVE about ongoing Annotation Studio work

### ISSUE THREAD

**Issue by daverett** [SHOREdevelopers/COVE#28]
Show basic metadata (# of annotations, annotations by user) in the Course dashboard at COVE.

**Comment by jamiefolsom** [comment]
We'll have to discuss this @daverett. There is a restful web service backing AS for annotations. You can access that if you like, to create calculated/aggregated information about user activity. Arguably, this should be deferred until after launch, since it's not mission-critical.

**Comment by daverett** [comment]
Yes. Not a high priority for now, though in some ways this would be the “face” of integration for a lot of users.

**Comment by jamiefolsom** [comment]
It would be great, but in the absence of some other features, like the edition builder, it's icing without cake. 😄
